newpr.py

```diff
@@ -205,6 +206,15 @@ def get_page_content(self, url):
         except urllib2.URLError:
             return None
 
+    def get_issues(self, labels):
+        suffix = ('+label:' + '+label:'.join(labels)) if labels else ''
+        result = self.api_req("GET", self.issues_url % (self.owner, self.repo) + suffix)
+        return result['items']
+
+    def get_issue_titles(self, labels):
+        issue_json = self.get_issues(labels)
+        map(lambda issue: (issue['number'], issue['title']), issue_json)
```

Comment:
Python doesn't have implicit returns. Looks like this should live in the base `APIProvider`, since it doesn't rely on any behaviour that needs to be mocked in tests.